### PR TITLE
Fix/chrome toggle

### PR DESCRIPTION
**Pull request by ahmaxed**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by ahmadabdolsaheb** [comment]
@ValeraS I removed the webkit-appearance instances as you suggested, does it look good enough?

**Comment by raisedadead** [comment]
That fixes it on my end. Nice work @ahmadabdolsaheb !! 🎉 

There are several other things (that are pre-exsting) that we need to fix (in separate PRs of course):
- The fonts on production and master do not match.
- The underline text decoration on links and the buttons should go away.
- The button sizes are different on production and master.
- The font size and weights are different on production and master

**Review by raisedadead — APPROVED** [review]

**Comment by ahmadabdolsaheb** [comment]
It was all @ValeraS but thanks. I took note of the issues and I will open an issue to deal with them accordingly.

File: client/src/components/layouts/layout.css [hunk 1]
```diff
@@ -122,12 +122,6 @@ button,
 select {
   text-transform: none;
 }
-[type='reset'],
-[type='submit'],
-button,
-html [type='button'] {
-  -webkit-appearance: button;
-}
```
Comment (ValeraS) [inline client/src/components/layouts/layout.css]: Let's remove only this and others not.

**Comment by ahmadabdolsaheb** [comment]
@ValeraS the change has been implemented.

**Pull request closed without merging**